pkg/aks/components/Config.vue

```diff
@@ -255,6 +250,7 @@ export default defineComponent({
         locationRequired:        requiredInCluster(this, 'aks.location.label', 'config.location'),
         resourceGroupRequired:   requiredInCluster(this, 'aks.clusterResourceGroup.label', 'config.resourceGroup'),
         dnsPrefixRequired:       requiredInCluster(this, 'aks.dnsPrefix.label', 'config.dnsPrefix'),
+        virtualNetworkRequired:  requiredInCluster(this, 'aks.virtualNetwork.label', 'config.virtualNetwork'),
```

Comment:
Below this line `ipv4WithOrWithoutCidr` is being passed 3 arguments even though the method only expects one argument.

It's not a part of this pr. and I can't comment on it in github so feel free to ignore but I figured I'd mention it since I saw it while browsing the code.